CMakeLists.txt

```diff
@@ -182,10 +182,7 @@ endif()
 
 # -----------------------------------------------------------------------------
 if(NGEN_WITH_NETCDF)
-    set(NETCDF_CXX "YES")
-    set(NETCDF_F77 "NO")
-    set(NETCDF_F90 "NO")
-    find_package(NetCDF REQUIRED)
+    find_package(NetCDF REQUIRED COMPONENTS CXX)
```

Comment:
Should probably require the C and CXX components at a minimum.

Reply:
The C library isn't a component, it's the base module, so if you call `find_package(NetCDF)` that only gets the C library.